src/main/java/bio/overture/ego/controller/TokenController.java

```diff
@@ -98,9 +98,12 @@ public TokenController(@NonNull TokenService tokenService) {
     return format("Token '%s' is successfully revoked!", token);
   }
 
-  @ResponseBody
-  List<TokenResponse> listTokens(@RequestHeader(value = "Authorization") String authorization) {
-    return null;
+  @RequestMapping(method = RequestMethod.GET, value = "/list_token")
```

Comment:
I think you are missing the annotation `@AdminScoped` to make sure only an admin user or application can list this.